### DISCUSSION TITLE
Masonry Image Layout

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#9778] [Feature Requests]
This could be a new Layout, or be a UI option of the Card Layout. Either way, the goal would be to have a more image-focused GUI with this styling: https://dribbble.com/shots/11937194-Hohor-Portfolio-Gallery

* Less padding / white-space
* Images can efficiently be shown in the correct aspect ratio (no cropping)
* Clean look, with no titles/subtitles until hover

**Comment by rijkvanzanten** [comment]
Can't wait for CSS Grid Masonry https://caniuse.com/mdn-css_properties_masonry 🤞🏻

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
